be/src/exprs/vectorized/bitmap_functions.cpp

```diff
@@ -332,4 +332,31 @@ ColumnPtr BitmapFunctions::bitmap_to_array(FunctionContext* context, const starr
     }
 }
 
+ColumnPtr BitmapFunctions::bitmap_max(FunctionContext* context, const starrocks::vectorized::Columns& columns) {
+    ColumnViewer<TYPE_OBJECT> viewer(columns[0]);
+
+    size_t size = columns[0]->size();
+    ColumnBuilder<TYPE_BIGINT> builder(size);
+    for (int row = 0; row < size; ++row) {
+        int64_t value = viewer.is_null(row) ? std::numeric_limits<uint64_t>::lowest() : viewer.value(row)->max();
```

Comment:
If the input is null, then it may be a better behaviour for the output to also be null
@Dshadowzh